scripts/migrate_spam.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import sys
+
+from modularodm import Q
+from website.app import init_app
+from website.project.model import Comment
+from framework.transactions.context import TokuTransaction
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+
+def migrate_status(records):
+    for record in records:
+        if len(record.reports) > 0:
+            record.spam_status = Comment.FLAGGED
+        else:
+            record.spam_status = Comment.UNKNOWN
```

Comment:
Is there a difference between `spam_status=UNKNOWN` vs `spam_status=None`? Would it be better if `Comment.UNKNOWN=None`?